FhirToDataLake/docs/Data-Mapping.md

```diff
@@ -15,74 +15,74 @@ Below is a JSON example that shows how the data is stored in Parquet.
 
 ### Wrap raw JSON fields into single string
 
-First, wrap some of the fields in raw JSON data into a single string in the Parquet file. You can still use [JSON functions](https://docs.microsoft.com/en-us/sql/t-sql/functions/json-functions-transact-sql?view=sql-server-ver15) to parse and analyze them on Synapse. 
+First, wrap below kind of fields in raw JSON data into a single string in the Parquet file. You can still use [JSON functions](https://docs.microsoft.com/en-us/sql/t-sql/functions/json-functions-transact-sql?view=sql-server-ver15) to parse and analyze them on Synapse workspace. 
```

Comment:
'below kind of' -> 'the following'
analyze them 'on Synapse workspace'-> analyze them "with Synapse Serverless Pool"

Reply:
Updated, thanks.